src/main/java/com/coveros/selenified/selenium/element/Excludes.java

```diff
@@ -28,41 +31,32 @@
     private static final String ONLYVALUE = ", only the values <b>";
     private static final String CLASSVALUE = " has a class value of <b>";
 
-    public Excludes(Action action, OutputFile file) {
-        this.action = action;
-        this.file = file;
-    }
-
-    public void setAction(Action action) {
-        this.action = action;
-    }
-
-    public void setOutputFile(OutputFile file) {
+    public Excludes(Element element, OutputFile file) {
+        this.element = element;
         this.file = file;
+        this.helper = new Helper(element, file);
     }
 
-    ///////////////////////////////////////////////////
-    // Our actual full implementation of the inherited overloaded methods
-    ///////////////////////////////////////////////////
+    // ///////////////////////////////////////
+    // assessing functionality
+    // ///////////////////////////////////////
 
     /**
      * checks to see if an element does not contain a particular class
      *
-     * @param element
-     *            - the element to be assessed
      * @param unexpectedClass
      *            - the unexpected class value
      * @return Integer: 1 if a failure and 0 if a pass
      */
-    public int classs(Element element, String unexpectedClass) {
+    public int classs(String unexpectedClass) {
```

Comment:
Consider renaming this method. Instead of the word classs I like clazz, though I like a descriptive name better